assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -126,6 +126,9 @@ export class BrightMirrorFiction extends Component<LocalBrightMirrorFictionProps
 
   componentWillReceiveProps(nextProps: LocalBrightMirrorFictionProps) {
     // Sync state
+    const { loading } = nextProps.brightMirrorFictionData;
```

Comment:
use static getDerivedStateFromProps(nextProps, prevState) instead of componentWillReceiveProps...